test/extended/default_mysql_test.go

```diff
@@ -0,0 +1,49 @@
+// +build default
```

Comment:
If I change this some something else than default, the go file is being ignored when building, so the test case is not executed.